src/get_primary_sam.py

```diff
@@ -224,6 +227,8 @@ def primary_and_unaligned_chimeric(sam_alnm_file, prefix, metagenome_list=None,
     :returns: an unaligned_len list, and strandness information
     """
 
+    is_trans = False
+
```

Comment:
Hey @kmnip , would you elaborate a bit more on this bug fix? I assume it only affected the "genome" mode with chimeric detection enabled. Am I right? There were no effects on metagenome and transcriptome modes, correct?

Reply:
The code crashes at line 408 when `is_trans` wasn't already defined.
`is_trans` was defined on line 235 only when `metagenome_list` was also defined as a list of metagenome species or `tpm`.
To fix this bug, `is_trans` must be pre-defined to `False`.
I think this bug can also affect the transcriptome mode when chimera detection is turned on but quantification mode is off.

Reply:
Thanks for the clarification. I was curious because I didn't ran into any issues before when I was benchmarking the transcriptome mode.